packages/kit-bg/src/vaults/impls/btc/Vault.ts

```
@@ -800,24 +831,28 @@ export default class VaultBtc extends VaultBase {
         path,
+        confirmations,
       }),
     );
 
     let outputsForCoinSelect: IOutputsForCoinSelect = [];
 
     if (isBatchTransfer) {
       outputsForCoinSelect = transfersInfo.map(({ to, amount }) => ({
+        type: EOutputsTypeForCoinSelect.Payment,
         address: to,
         value: parseInt(
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)